lms/djangoapps/grades/tasks.py

```diff
@@ -7,24 +7,27 @@
 from django.contrib.auth.models import User
 
 from lms.djangoapps.course_blocks.api import get_course_blocks
-from lms.djangoapps.courseware.courses import get_course_by_id
 from opaque_keys.edx.keys import UsageKey
 from opaque_keys.edx.locator import CourseLocator
 from openedx.core.djangoapps.content.block_structure.api import get_course_in_cache
+from xmodule.modulestore.django import modulestore
 
 from .config.models import PersistentGradesEnabledFlag
 from .transformer import GradesTransformer
 from .new.subsection_grade import SubsectionGradeFactory
 
 
 @task(routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id):
+def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher):
```

Comment:
Let me know if you need help with the merge dancing that I expect will be needed here, now that my recent task-related work has landed.